.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+#### Changes
+
+- ...
+
+#### Checklist
+
+<!-- please check all items and add your own -->
+
+- [x] Read the contribution [guide](../CONTRIBUTING.md) and accept the
+  [code](../CODE_OF_CONDUCT.md) of conduct
+- [ ] Readme (updated or not needed)
+- [ ] Tests (added, updated or not needed)
```

Comment:
Might be good to add a checkbox for adding to the documentation repo, like what is in Unity builder.

Reply:
Added it. Albeit in a slightly more concise way, as with these things less is more.